IpmiFeaturePkg/Docs/Ipmi_Feature.md

```diff
@@ -32,8 +37,8 @@ __Generic IPMI__ - Implements the generic IPMI in a protocol, PPI, or MM driver.
 This layer is responsible for taking a high level IPMI request and building the
 IPMI request and parsing the response. Generic IPMI will rely on the transport
 library to handle the hardware specifics. External consumers of the IPMI package
-should avoid directly calling into the IPMI generic interface and should instead
-use the base library mentioned below. The generic interface is defined in the
+should consider uring the base library instead of directly calling the protocols.
+The generic interface is defined in the

```

Comment:
generic interface....do you mean the Ipmi Transport interface? 

Maybe something like "Even though the Ipmi transport interface is defined in ../Include/IpmiInterface.h, this is considered an internal protocol. Which is why it is not defined in the Include/Protocols direcotry. External users are asked to use the IpmiBaseLibs instead."